cli/src/test/java/org/jboss/as/cli/completion/operation/test/OperationNameCompletionTestCase.java

```diff
@@ -44,6 +46,8 @@
 
     private MockCommandContext ctx;
     private OperationRequestCompleter completer;
+    private CommandRegistry cmdRegistry;
+    private CommandCompleter cmdCompleter;
```

Comment:
I am not sure that this test should be extended with a test for CommandCompleter. You should possibly create a new CommandCompleterTestCase with some tests to cover specifically the CommandCompleter for:
- operations
-commands
and multilines.
Thank-you.